railties/lib/rails/generators/rails/plugin/templates/rubocop.yml.tt

```diff
@@ -1,6 +1,9 @@
 # Omakase Ruby styling for Rails
 inherit_gem: { rubocop-rails-omakase: rubocop.yml }
 
+AllCops:
+  TargetRubyVersion: <%= ENV["RBENV_VERSION"] || ENV["rvm_ruby_string"] || RUBY_ENGINE_VERSION %>
```

Comment:
Should we add this as `required_ruby_version` in the plugin's gemspec since Rubocop may suggest using newer syntax?